src/resolvers.ts

```diff
@@ -16,6 +17,16 @@ import {
 
 const resolvers = {
     Query: {
+        degrees: () => {
```

Comment:
let's talk through this one. 
I'm thinking that degrees would always be queried with healthcare provider data. 

If that makes sense, 
then we should probably use your degree query within the `healthcareProfessionalService` to build the aggregated data. 

if we think degrees need to be fetched separately, 
- what would be the use cases? 
- The queries look awesome and simple. ✨

Reply:
When a user submits new data, they may need to know what degree types are available

E.g. we have some form for adding a new HealthcareProvider and we want to show the types that should be added

But maybe I'm thinking from more of a REST perspective and the InputTypes give us what we need?

Also some of these simple queries are a good teaching example for folks learning graphQL and sql, so I thought they could help us establish patterns. 🤔 but might not want to overdo it